include/materials/modified_cam_clay.tcc

```diff
@@ -568,8 +568,6 @@ Eigen::Matrix<double, 6, 1> mpm::ModifiedCamClay<Tdim>::compute_stress(
   const double p_trial = (*state_vars).at("p");
   // Deviatoric trial stress
   const double q_trial = (*state_vars).at("q");
-  // Lode angle of trial stress
-  const double theta_trial = (*state_vars).at("theta");
```

Comment:
@tianchiTJ, just making sure, this is not needed and can be deleted right?

Reply:
This is inside the `compute_stress` function call not in the creation of variables. This is not being used in this function.